vips/vip-400/bscmainnet.ts

```diff
@@ -0,0 +1,156 @@
+    [
+      // Configure Oracle
+      {
+        target: REDSTONE_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[marketSpec.vToken.underlying.address, SOLVBTC_REDSTONE_FEED, SOLVBTC_MAX_STALE_PERIOD]],
+      },
+
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            marketSpec.vToken.underlying.address,
+            [REDSTONE_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+          ],
+        ],
+      },
+
+      // Add Market
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_supportMarket(address)",
+        params: [marketSpec.vToken.address],
+      },
+
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketSupplyCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.supplyCap]],
+      },
+
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.borrowCap]],
+      },
+
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setCollateralFactor(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.riskParameters.collateralFactor],
+      },
+
+      {
+        target: marketSpec.vToken.address,
+        signature: "setAccessControlManager(address)",
+        params: [ACCESS_CONTROL_MANAGER],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setProtocolShareReserve(address)",
+        params: [PROTOCOL_SHARE_RESERVE],
+      },
+
+      {
+        target: marketSpec.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [REDUCE_RESERVES_BLOCK_DELTA],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "_setReserveFactor(uint256)",
+        params: [marketSpec.riskParameters.reserveFactor],
+      },
+
+      // Mint initial supply
+      {
+        target: VTREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [marketSpec.vToken.underlying.address, marketSpec.initialSupply.amount, NORMAL_TIMELOCK],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "mintBehalf(address,uint256)",
+        params: [marketSpec.initialSupply.vTokenReceiver, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, 0],
+      },
+    ],
```

Comment:
We need commands to configure the token converters

Reply:
https://github.com/VenusProtocol/vips/pull/407/commits/5b8e5c64511b46799d2cfa3c7429fc6124eee510